docs/src/examples/components/Label/Variations/index.tsx

```diff
@@ -6,28 +6,28 @@ const Variations = () => (
   <ExampleSection title="Variations">
     <ComponentExample
       title="Circular"
-      description="A label can be circular"
+      description="A label can be circular."
       examplePath="components/Label/Variations/LabelExampleCircular"
     />
     <ComponentExample
       title="Icon"
-      description="The label can contain an icon"
+      description="The label can contain an icon."
       examplePath="components/Label/Variations/LabelExampleIcon"
     />
     <ComponentExample
       title="Icon position"
-      description="The icon inside the label can be aligned before or after the content"
+      description="The icon inside the label can be aligned before or after the content."
       examplePath="components/Label/Variations/LabelExampleIconPosition"
     />
     <ComponentExample
       title="Clickable icon"
-      description="The icon inside the label can be clickable"
+      description="The icon inside the label can be clickable."
       examplePath="components/Label/Variations/LabelExampleOnIconClick"
     />
     <ComponentExample
-      title="The Icon inside Label can be customized"
-      description="The Icon component inside the Label can be defined with customizing it's prop"
-      examplePath="components/Label/Variations/LabelExampleIconAsShorthand"
+      title="Icon props"
```

Comment:
Do not put sentences in example titles.  Titles should be prop names.  If no prop name is available then use a very concise title.